go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/prometheus/client_golang v1.10.0
 	github.com/rdegges/go-ipify v0.0.0-20150526035502-2d94a6a86c40
 	github.com/submariner-io/admiral v0.9.0-m2
-	github.com/submariner-io/shipyard v0.9.0-m2
+	github.com/submariner-io/shipyard v0.9.0-m2.0.20210406162037-044072e6a6ce
```

Comment:
Is it OK to make this change this way? Or should we just wait for next shpyard release?

Reply:
It's fine :-)